src/scss/_mixins.scss

```diff
@@ -96,7 +96,9 @@
 				$name: "#{$color}-#{$value}";
 				$tint: null;
 
-				@if $saturation == null {
+				@if $saturation == null and $color == 'internal-slate' {
```

Comment:
Instead of this, what if `oColorsMix` had a default `$background` of `white` for the internal brand?
https://github.com/Financial-Times/o-colors/blob/814daaee78c9e52ea83d1cdbe66f7e0e5bd0797e/src/scss/_functions.scss#L28

`oBrandGet('mix-color')`?

Reply:
:+1: As discussed, issue: #144